integration_tests/payloads/snsEventPayload.ts

```diff
@@ -93,5 +93,5 @@ export {
   snsEventMissingTimestampPayload,
   snsEventWithAdditionalFieldsPayload,
   snsMissingEventNamePayload,
-  snsEventMisisingEventIdValue,
+  snsEventMisisingEventIdValue,generateRandomNumber
```

Comment:
Top of  out January task list has to be getting your linter playing nice with your editor 😅

Reply:
yes need to make ESLint to work in Editor